test/unit_tests/cuda/unit_test_image_function_cuda.cpp

```diff
@@ -207,8 +207,8 @@ namespace image_function_cuda
             const std::vector < uint8_t > intensity = intensityArray( 2 );
             PenguinV_Image::Image input = uniformImage( intensity[0], 0, 0, reference );
 
-            const bool horizontalFlip = (randomValue<uint32_t>( 0, 2 ) == 0);
-            const bool verticalFlip = (randomValue<uint32_t>( 0, 2 ) == 0);
+            const bool horizontalFlip = ( randomValue<uint32_t>( 0, 1 ) == 0 );
```

Comment:
Let's create **randomBoolValue()** function instead. No harm to do this but it makes code easier to read.

Reply:
For this function we could use `rand()` directly, no need to call `randomValue<>` function